api/services/userService.js

```diff
@@ -0,0 +1,86 @@
+const { userDao } = require('../models');
+const jwt = require('jsonwebtoken');
+const axios = require('axios');
+const qs = require('qs');
+
+const signIn = async (code) => {
+  let ACCESS_TOKEN;
+  let userInformation;
+
+  await axios
+    .post(
+      `https://nid.naver.com/oauth2.0/token`,
+      qs.stringify({
+        grant_type: 'authorization_code',
+        client_id: 'M7RqsEOvkI7Fb33qpPnq',
+        client_secret: 'p87VetJeBY',
+        state: 'cosegu',
+        code: code,
+      }),
+      {
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+        },
+      }
+    )
+    .then((response) => {
+      ACCESS_TOKEN = response.data.access_token;
+    });
+
+  await axios
+    .get('https://openapi.naver.com/v1/nid/me', {
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+        Authorization: `Bearer ${ACCESS_TOKEN}`,
+      },
+    })
+    .then((res) => {
+      const { response } = res.data;
+      userInformation = response;
+    });
```

Comment:
```suggestion
  const kakaoAccessToken = await axios
    .post(
      `https://nid.naver.com/oauth2.0/token`,
      qs.stringify({
        grant_type: 'authorization_code',
        client_id: 'M7RqsEOvkI7Fb33qpPnq',
        client_secret: 'p87VetJeBY',
        state: 'cosegu',
        code: code,
      }),
      {
        headers: {
          'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
        },
      }
    ).data.access_token;

  const userInfo = await axios
    .get('https://openapi.naver.com/v1/nid/me', {
      headers: {
        'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
        Authorization: `Bearer ${ACCESS_TOKEN}`,
      },
    }).data.response;
```
이렇게 작성하시면 콜백으로 작성하는 것보다 가독성이 더 좋습니다. 자세한 네이밍은 직접 테스트해보시면서 수정해 주세요!